cmd/start.go

```diff
@@ -117,11 +134,37 @@ func init() {
 		"",
 		"[OPTIONAL] Path to logfile (default stdout)")
 
+	startCmd.Flags().StringVar(
+		&tunnelServerPort,
+		"tunnel-port",
+		"5223",
+		"[OPTIONAL] Address on which http tunnel server is running")
+
+	startCmd.Flags().StringVar(
+		&tunnelPortRange,
+		"tunnel-port-range",
+		"20000:30000",
+		"[OPTIONAL] Range of ports which is used to open http tunnels")
+
 	RootCmd.AddCommand(startCmd)
 }
 
 func startCommand(_ *cobra.Command, _ []string) {
 	DisplayBanner()
+
+	var tunnelURL string
+	if publicIP == "" {
+		IP, err := ip.Get()
+		if err != nil {
+			log.Error("Unable to fetch public IP ", err)
```

Comment:
```suggestion
			log.Error("Unable to fetch public IP. Please set one explicitly", err)
```
You could put that as fatal log